spec/features/user_creates_orders_spec.rb

```diff
@@ -17,11 +17,11 @@
 
         user = User.create(username: "adam", password: "adam")
         d = Destination.create(continent: "North America", description: "North American continent description.",
-                            lat: 41.8369, long: -87.6847, image_url: "south-america.jpg" )
+                            image_url: "south-america.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.